jp2_pc/Source/Lib/Sys/FastHeap.hpp

```diff
@@ -779,10 +779,9 @@ class CFastHeap: private CDArray<char>
 
 	friend void* operator new(uint u_size_type, CFastHeap& fh_heap);
 	friend void* operator new(uint u_size_type, CFastHeap& fh_heap, uint u_alignment);
-#ifdef __MWERKS__
+
 	friend void* operator new [] (uint u_size_type,	CFastHeap& fh_heap);
 	friend void* operator new [] (uint u_size_type, CFastHeap& fh_heap, uint u_alignment);
-#endif
 };
 
```

Comment:
Trespasser very often treats `new`/`new[]` and `delete`/`delete[]` as the same, but they are not.